nessie/sanity_api_test.go

```diff
@@ -121,7 +121,8 @@ func TestSanityAPI(t *testing.T) {
 	diffResp, err = client.Refs.DiffRefs(refs.NewDiffRefsParamsWithContext(ctx).
 		WithRepository(repo).
 		WithLeftRef("branch1").
-		WithRightRef(masterBranch), nil)
+		WithRightRef(masterBranch).
```

Comment:
Sorry, have nothing to say about last three

Reply:
Good - there are the changes needed to make the new API changes work